src/Routes/Devices/UpdateSystem.js

```diff
@@ -211,7 +211,7 @@ const UpdateSystemMain = ({
     const pathLen = destPath.split('/').length;
     const dest = pathname.split('/').slice(0, pathLen).join('/');
     if (navigateProp) {
-      navigate({ ...dest });
+      navigate(paths.insightsInventory);
```

Comment:
think this will not work in system detail view

Reply:
its working in both system